Dockerfile

```diff
@@ -2,12 +2,10 @@ FROM node:16.16-alpine3.15
 
 WORKDIR /usr/src/app
 
-COPY package*.json .
```

Comment:
Quick question : why did you remove the copy of `package*.json` ?  was it useless ?

Reply:
`COPY . .` will copy over all the files including the `packages*.json`. I needed to the move this statement before the  `RUN yarn install` because the `yarn install` needed files that were not present 👍